.devcontainer/devcontainer.json

```diff
@@ -61,6 +61,11 @@
 		"go.useLanguageServer": true,
 		"go.gopath": "/go"
 	},
+    "hostRequirements": {
```

Comment:
Is this strictly necessary?

The problem is that this will make Codespaces only use the largest VMs, which are expensive for our users unless they're really needed